src/testWithSpringBoot_3_0/java/org/openrewrite/java/spring/boot3/MigrateResponseEntityExceptionHandlerHttpStatusToHttpStatusCodeTest.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright 2024 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.spring.boot3;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.DocumentExample;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+class MigrateResponseEntityExceptionHandlerHttpStatusToHttpStatusCodeTest implements RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec.recipe(new MigrateResponseEntityExceptionHandlerHttpStatusToHttpStatusCode())
+          .expectedCyclesThatMakeChanges(2)
```

Comment:
This is usually an indication that we're not making changes in the right order, or matching twice on the same elements. If you can't work it out then I'll need to dive in for a fix.

Reply:
Recipe now completes in 1 cycle :)